### ISSUE TITLE
Broken ARIA references Vet Center locations list Content Type in CMS 

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15830]
## Describe the defect
Within the edit screens on the Vet Center locations list content type, there are multiple `aria-describedby` attributes that do not correspond to an ID anywhere on the page.
  - Display additional mobile vet centers
  - Nearby vet centers and outstations

### Discovery & Recommendations
- The `<details>` tag of the 'Display additional mobile vet centers' block has `aria-describedby="edit-field-nearby-mobile-vet-centers--description"`, but if I search for that ID I can't find it anywhere on the page, it doesn't exist. There is a description field in the details component, but that info can be accessed by opening the details and reading as usual, so [using aria is unnecessary](https://dsva.slack.com/archives/C8E985R32/p1761168017435559).
  - [ ] Remove the `aria-describedby` on the `<details>` tag.
- The `<details>` tag of the 'Nearby vet centers and outstations' block has `aria-describedby="edit-nearby-vc-information--description"`, but if I search for that ID I can't find it anywhere on the page, it doesn't exist. There is a description field in the details component, but that info can be accessed by opening the details and reading as usual, so [using aria is unnecessary](https://dsva.slack.com/archives/C8E985R32/p1761168017435559).
  - [ ] Remove the `aria-describedby` on the `<details>` tag.

### AC / Expected behavior
- [ ] The following fields/sections no longer throw errors about unlinked aria-describedby attributes:
  - [ ] Display additional mobile vet centers
  - [ ] Nearby vet centers and outstations
- [ ] Accessibility review

## Screenshots
Screenshot of the Display additional mobile vet centers field error from AMP plugin. The `aria-describedby` attribute is pointing to something with the ID `edit-field-nearby-mobile-vet-centers--description` but if I search for that ID I can't find it anywhere on the page, it doesn't exist. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/7eb0af2c-69c7-40f6-8207-3216b78907f6)

Screenshot of where the attribute is on the `<div>`
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0357a52d-8a7d-4d1d-84f3-9d7609a43d95)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`
